dipy/align/parzenhist.pyx

```diff
@@ -708,10 +708,10 @@ cdef _compute_pdfs_dense_3d(double[:, :, :] static, double[:, :, :] moving,
         cnp.npy_intp offset, valid_points
         cnp.npy_intp k, i, j, r, c
         double rn, cn
-        double val, spline_arg, sum
+        double val, spline_arg, s
```

Comment:
same comment as above: We try to avoid 1 letter variable (apart from maths equation). Can you rename it? `total, summation, ...`

Reply:
Done in 4e9cab5.